pkgs/development/compilers/llvm/17/default.nix

```diff
@@ -215,6 +215,7 @@ in let
           (!stdenv.targetPlatform.isWasm && stdenv.targetPlatform.useLLVM or false)
           "-lunwind"
         ++ lib.optional stdenv.targetPlatform.isWasm "-fno-exceptions";
+      nixSupport.cc-ldflags = lib.optionals (!stdenv.targetPlatform.isWasm) [ "-L${targetLlvmLibraries.libunwind}/lib" ];
```

Comment:
shouldn't this be:
`(!stdenv.targetPlatform.isWasm && stdenv.targetPlatform.useLLVM or false)`?

Reply:
unneeded as  clangUseLLVM is only used when `stdenv.targetPlatform.useLLVM` is true.